public/distinctions.json

```diff
@@ -0,0 +1,23 @@
+{
+  "Paris Peace Forum": {
```

Comment:
I understand that order is defined in this file.
Let's switch to reverse chronological order (and avoid too many clicks on the most outdated Paris Peace Forum distinction):
1.  DPGA (most clicked)
2. Nobel Prize (most visible, in the middle)
3. Paris Peace Forum